h2o-core/src/test/java/water/fvec/FrameTest.java

```diff
@@ -43,7 +72,6 @@ public static void setup() {
       subset1.delete();
 //      subset2.delete();
       if (x != null) x.delete();
-      if (y != null) y.delete();
```

Comment:
I would not modify the ignored test... + the y deletion was removed, but not actual `y` creation (if somebody uncomment the code).